packages/expo-live-photo/expo-module.config.json

```
@@ -0,0 +1,6 @@
+{
+  "platforms": ["ios"],
```

`ios` is deprecated in favor of `apple` (which basically reads the supported platforms from the podspec)

```suggestion
  "platforms": ["apple"],
```